lib/schedule/minischedule/trip.ex

```diff
@@ -14,7 +14,9 @@ defmodule Schedule.Minischedule.Trip do
           via_variant: RoutePattern.via_variant() | nil,
           run_id: Run.id() | nil,
           start_time: Util.Time.time_of_day(),
-          end_time: Util.Time.time_of_day()
+          end_time: Util.Time.time_of_day(),
+          start_place: String.t(),
+          end_place: String.t()
```

Comment:
Looks like we have `Schedule.Hastus.Place.id()` for this.

Though after hydration, they aren't really ids anymore. I wonder if we should add a `Schedule.Hastus.Place.name :: String.t()` and have everywhere that hydrates an id into a name have their type be `Place.id() | Place.name()`.

Reply:
Ah heck, `Hastus.Place.id()` and `Gtfs.Timepoint.id()` are kind of tangled up. It may not be worth it to sort out right now.